### Assembly and Annotation

This version of the genome sequence of the PH-1 strain has been provided
by Rothamsted Research and the Centre for Genome Analysis and has been
submitted to the International Nucleotide Sequence Database Consortium
(INSDC) as [PRJEB5475](http://www.ebi.ac.uk/ena/data/view/PRJEB5475).
The dataset was produced by combining the draft version of the genome
previously generated by the Broad Institute using Sanger sequencing and
manually annotated[ by the Fungal and Microbial genomics group at the
Munich Information Services for Protein Sequences
(submitted to INSDC as
GCA_000240135](http://www.ebi.ac.uk/ena/data/view/GCA_000240135)[),
with additional whole shotgun data. In addition to filling gaps in the
previous assembly, the new version has: identified a mis-assembly in
chromosome 1 at the 5' end, and extended the AT rich regions, which
include the identification for the first time of telomere and centromere
sequences. This has increased the genomic sequence by 484,290 bp. Four
previously unplaced supercontigs have been located within the new
assembly or within the mitochrondrial sequence. The remaining unlocated
supercontigs have been identified as repeat sequences, with all but one
matched in part to the repeat sequence at the 3' end of chromosome 4
(which has not been counted in the estimate of the total genome length).
In addition to the chromosomal updates, a revised mitochondrial sequence
has also been produced.

Non coding RNA genes have been annotated using tRNAScan-SE (Lowe, T.M.
and Eddy, S.R. 1997), RFAM (Griffiths-Jones et al 2005), and RNAmmer
(Lagesen K.,et al 2007); additional analysis tools have also been
applied.
